src/components/mainLayout/footer/footer.tsx

```diff
@@ -2,29 +2,22 @@
 
 import Link from "next/link";
 
-import { useWindowWidth } from "@react-hook/window-size";
-
-import cn from "classnames";
-
-import { usePathname } from "next/navigation";
-
 import { MainPath } from "../../../services/router/routes";
 
 import styles from "./footer.module.scss";
 
-const Footer = () => {
-	const pathname = usePathname();
-	const currentWindowWidth = useWindowWidth();
-	const windowWidth = 1024;
+interface IFooterProps {
+	isMainPage?: boolean;
+}
+
```

Comment:
can you move this interface in src/types/common/ComponentsProps.ts with name of IFooter?